Sources/Core/Shared/ComposedOperation.swift

```diff
@@ -18,7 +18,7 @@ public class ComposedOperation<T: Operation>: Operation {
     }
 
     init(operation composed: T) {
-        self.target = composed as? Operation ?? GroupOperation(operations: [composed])
+        self.target = composed ?? GroupOperation(operations: [composed])
```

Comment:
Yeah, sorry - wasn't clear, the `danthorpe:swift/3.0` branch has the error - it's why it's not really ready yet. Basically, this is because the name clash causes Xcode's migrator to produce incorrect, but compiling changes. I showed this to the Xcode/Swift team at WWDC.

I will focus on the Swift 3.0 branch this weekend. Basically I'm going to go through the changes an double check every `Operation` instance to verify if it should be `Foundation.Operation` or `Operations.Operation`.